did_resolver_sov/src/reader/vdr_reader.rs

```diff
@@ -28,13 +28,13 @@ impl TryFrom<LedgerPoolConfig> for ConcreteAttrReader {
             .capacity(1000)?
             .build();
         let response_cacher = Arc::new(InMemoryResponseCacher::new(cacher_config));
-        let config = IndyVdrLedgerConfig {
+        let config = IndyVdrLedgerReadConfig {
             request_signer,
```

Comment:
I tried pulling the branch locally and I'm getting an error from cargo check that there's no `request_signer` in the struct. How did the CI pass though? Am I doing something wrong? As it does seem that the field was indeed removed from the struct.